### DISCUSSION TITLE
I created Terraform script for  HA cluster setup using own VPC

### DISCUSSION THREAD

**Discussion by FathiQ** [gravitational/teleport#31395] [Show and tell]
Hi all, 

already Teleport has Terraform to deploy the Teleport HA cluster on AWS but the problem is it will create a new VPC which is not the case most of the time.

I already created it and I want to do PR.

let me know if you want a demo before PR.

**Comment by FathiQ** [comment]
https://github.com/gravitational/teleport/pull/31394

**Reply by webvictim** [reply]
Thank you for this. It's unlikely that we'll accept the PR due to avoid fragmenting our documented Terraform setup methods, but I appreciate the effort you went to and I'm glad it's useful for your own purposes.
